src/Compilers/VisualBasic/Portable/Compilation/SemanticModel.vb

```diff
@@ -2299,6 +2299,23 @@ _Default:
         ''' </summary>
         Public MustOverride Shadows ReadOnly Property ParentModel As SemanticModel
 
+        ''' <summary>
+        ''' Determines what type of conversion, if any, would be used if a given expression was
+        ''' converted to a given type.
+        ''' </summary>
+        ''' <param name="expression">An expression which must occur within the syntax tree
+        ''' associated with this object.</param>
+        ''' <param name="destination">The type to attempt conversion to.</param>
+        ''' <param name="isExplicitInSource">Unused.</param>
+        ''' <returns>Returns a Conversion object that summarizes whether the conversion was
+        ''' possible, and if so, what kind of conversion it was. If no conversion was possible, a
+        ''' Conversion object with a false "Exists " property is returned.</returns>
+        ''' <remarks>To determine the conversion between two types (instead of an expression and a
+        ''' type), use Compilation.ClassifyConversion.</remarks>
+        Public Overrides Function ClassifyCommonConversion(expression As SyntaxNode, destination As ITypeSymbol, Optional isExplicitInSource As Boolean = False) As CommonConversion
+            Return ClassifyConversion(DirectCast(expression, ExpressionSyntax), destination).ToCommonConversion()
```

Comment:
`isExplicitInSource` is not used - is there a parameter missing to ClassifyConversion? (and this doesn't match C# version)